### ISSUE TITLE
Standing on plating between PA room and Activated Containment field is death

### ISSUE THREAD

**Issue by Genesis720** [yogstation13/yogstation#1785]
This bug killed 3 engineers tonight, starting with me, the CE, and then the two who tried to rescue me.

PA is wired and set up, containment field is activated, emitters are running. The Energy beams between all field generators are up, but the PA is not on yet. As the CE, I am making a full circuit of the containment field just to check it before I go to turn the PA on. As I am walking on the plating that runs directly between the PA room and the EM field (one tile wide, with PA room glass to the north and EM field to the south), I am pulled into the field, as though a singularity was running and pulling things in. The first one doesn't kill me, so I stand up, make sure my magboots were activated (they had been the whole time) and attempt to step to the right. Same thing happens, I am thrown into the EM field. Once I stand back up, I switch from run to walk mode, and try and move left. Again I am pulled into the EM field. My next attempt to move parallel to the EM field puts me in crit.

This same "effect" (bug?) killed 2 more engineers who tried to help me. Made a ticket, and was told to file a bug report because it killed so many of us.

TL;DR: attempting to move left or right on the plating between the PA room glass and the EM field, while EM field is up results in 3 engineers getting pulled towards the EM field with no way to escape. All 3 died.

**Comment by Genesis720** [comment]
Whatever was happening was triggering the containment field response: the spinning animation, burn damage and long knockdown. It may have been throwing me into the window, but since the window was to my immediate north, I couldn't tell. Once you stood back up, you were fine, unless you tried to move laterally in either direction (which was parallel to the containment field), which immediately triggered the burn damage, spinning animation and knockdown.
